src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private final DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+    private final DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("MMM dd yyyy, HH:mm");
+    protected LocalDateTime time;
+
+    public Deadline(String description, String time) {
+        super(description);
+        this.time = LocalDateTime.parse(time, inputFormatter);
```

Comment:
It would have been better if Deadline could handle the `InvalidParseException` which would occur when the date entered is in the incorrect format. 

By handling this exception, you can prompt the user to use the correct format in the message.